src/mesh.cpp

```diff
@@ -5,6 +5,9 @@
 #include <string>
 #include <gsl/gsl>
 
+#include <sstream>
+#include <fstream>
```

Comment:
Are these new headers being used? I don't see other changes in this file that use them.